src/webgpu/web_platform/reftests/canvas_colorspace_bgra8unorm.https.html

```diff
@@ -14,6 +14,7 @@
   <link rel="help" href="https://gpuweb.github.io/gpuweb/" />
   <meta name="assert" content="WebGPU bgra8norm canvas with colorSpace set should be rendered correctly" />
   <link rel="match" href="./ref/canvas_colorspace-ref.html" />
+  <meta name=fuzzy content="maxDifference=0-2;totalPixels=0-99999">
```

Comment:
Actually these tests render a very large area (twelve 128x128 canvases, possibly times the devicePixelRatio^2) so I'm going to increase these.